### DISCUSSION TITLE
How to arrange the PODs release sequence of deployment resources? Does ArgoCD have support for this feature?

### DISCUSSION THREAD

**Discussion by peter-hst** [argoproj/argo-cd#11070] [Q&A]
Hello, I would like to ask a question, can ArgoCD's advice on deploying PODs specify the order between them?

How to arrange the PODs release sequence of deployment resources? Does ArgoCD have support for this feature?

**Comment by riyas-rawther** [comment]
This would be nice. When we deploy Harbor using Helmchart, the Job service pod get degraded due to the pending deployment of Core service.
